src/core/public/chrome/ui/header/_index.scss

```diff
@@ -1,5 +1,11 @@
 @include euiHeaderAffordForFixed;
 
+.euiDataGrid__restrictBody {
```

Comment:
This is necessary to prevent the overlapping of our application header with the full height view of data grid